packages/e2e/src/Util/mnemonic.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import { Cardano } from '@cardano-sdk/core';
+import { KeyManagement } from '@cardano-sdk/wallet';
+
+/**
+ * Generates a new set of Mnemonic words and prints them to the console.
+ */
+(async () => {
+  let mnemonic = '';
+  const mnemonicArray = KeyManagement.util.generateMnemonicWords();
+  for (const word of mnemonicArray) mnemonic += `${word} `;
+
+  const keyAgentFromMnemonic = await KeyManagement.InMemoryKeyAgent.fromBip39MnemonicWords({
+    getPassword: async () => Buffer.from(''),
```

Comment:
It would be great to replace the unencrypted mnemonic with an encrypted one and use an encrypted one in `.env`. We could then do something like `yarn generate-mnemonic [password]` and print both encrypted and unencrypted one. Out of scope for this PR, just an idea to have it as an example of good security practices.

Reply:
I can add the option of a password, I think it can be useful. 

But if we add the seeds encrypted, then the user would also need to provide the password they chose on the .env file

Reply:
Captured in ADP-1972